packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -15,6 +20,23 @@ const variants = {
   DOCUMENT: 'document',
 };
 
+const getHyperlinkIcon = (variant) => {
+  switch (variant) {
+    case variants.AUDIO:
+      return (<span className={cx('icon')}><IconAudio /></span>);
+    case variants.DOCUMENT:
+      return (<span className={cx('icon')}><IconDocuments /></span>);
+    case variants.EXTERNAL:
+      return (<span className={cx('icon')}><IconExternalLink /></span>);
+    case variants.IMAGE:
+      return (<span className={cx('icon')}><IconImage /></span>);
+    case variants.VIDEO:
+      return (<span className={cx('icon')}><IconVideoCamera /></span>);
```

Comment:
The span can probably be moved out of this switch statement and used mutually below so it's not redeclared for each icon. 

```jsx
<ComponentType>
  {children}
  <span className={cx('icon')}>
    {getHyperlinkIcon(variant)}
  </span>
</ComponentType>
```

Reply:
That's right. I'd keep it as-is.